augur/tasks/github/facade_github/contributor_interfaceable/contributor_interface.py

```diff
@@ -406,18 +407,18 @@ def get_login_with_commit_hash(session, commit_data, repo_id):
 
 
 
-def create_endpoint_from_repo_id(session, repo_id):
+def create_endpoint_from_repo_id(manifest, repo_id):
```

Comment:
This function should be changed, so it takes `augur_db`, `logger`, and `repo_id` as arguments. The manifest class is not made to be passed around, it is made to be a utility class that makes it easier to create all the needed objects for a github task and that all those objects are closed properly.